playground/infrastructure/ci_cd.py

```diff
@@ -43,6 +43,12 @@
     "beam examples/tests/katas and their outputs on the GCD",
     choices=[Config.CI_STEP_NAME, Config.CD_STEP_NAME],
 )
+parser.add_argument(
```

Comment:
@TSultanov  could you please update playground/README.md with info regarding new script argument